spring-cloud-gcp-spanner/src/main/java/org/springframework/data/spanner/core/mapping/BasicSpannerPersistentEntity.java

```diff
@@ -0,0 +1,90 @@
+/*
+ *  Copyright 2017 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.data.spanner.core.mapping;
```

Comment:
I think this wrong package in this project. Must be `org.springframework.gcp.data` I guess.
Also we are in `2018` already. The Copyright must be actual, please

Reply:
Fixed